internal/network/command.go

```diff
@@ -241,3 +241,30 @@ func (c *command) addPrivateLinkAttachmentFlag(cmd *cobra.Command) {
 	cmd.Flags().String("attachment", "", "Private link attachment ID.")
 	pcmd.RegisterFlagCompletionFunc(cmd, "attachment", c.validPrivateLinkAttachmentArgsMultiple)
 }
+
+func addAcceptedNetworksFlag(cmd *cobra.Command, c *pcmd.AuthenticatedCLICommand) {
+	cmd.Flags().StringSlice("accepted-networks", nil, "A comma-separated list of accept policy networks.")
```

Comment:
```suggestion
	cmd.Flags().StringSlice("accepted-networks", nil, "A comma-separated list of accepted policy networks.")
```

Reply:
<img width="456" alt="image" src="https://github.com/confluentinc/cli/assets/146779173/0f3335e8-e1b8-4be8-b4b1-a5b2d112eb81">


This refers to the “Accept policy" but wording wasn’t great, its been updated to:

> A comma-separated list of networks from which connections can be accepted.